pkgs/servers/sql/postgresql/plv8/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitHub, v8, perl, postgresql }:
+
+stdenv.mkDerivation rec {
+  name = "plv8-${version}";
+  version = "2.0.3";
+
+  nativeBuildInputs = [ perl ];
+  buildInputs = [ v8 postgresql ];
+
+  src = fetchFromGitHub {
+    owner = "plv8";
+    repo = "plv8";
+    rev = "v${version}";
+    sha256 = "0cn7ynckmdb08dkzjilvc55xz61d1jiya7yrnphizw404j84y3qc";
+  };
+
+  preConfigure = ''
+    substituteInPlace Makefile --replace '-lv8_libplatform' '-lv8_libplatform -lv8_libbase'
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin
```

Comment:
yes, all postgresql plugins have empty ```$out/bin```
one of them has comment [current postgresql extension mechanism in nixos requires bin directory](https://github.com/NixOS/nixpkgs/blob/master/pkgs/servers/sql/postgresql/pgjwt/default.nix#L13)